barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.hpp

```diff
@@ -1,610 +1,26 @@
 #pragma once
 #include "acir_format.hpp"
-#include "barretenberg/common/container.hpp"
```

Comment:
Nearly all of the includes were for unused about half of the functions are not used externally. I thought moving the new function you @ledwards2225 wrote would help but in the end I still needed a macro--thought I'd keep the change to this header though since it's a good change.